packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -232,6 +242,7 @@ export const Headline = styled.h3`
   color: ${C_EBON};
   margin: 0; /* Reset */
   padding-bottom: ${GEL_SPACING};
+  ${({ displayImage }) => !displayImage && `display: inline;`}
```

Comment:
This `display: inline` is making the heading overlap the summary

![bug](https://user-images.githubusercontent.com/46446236/67271775-23e45280-f4b3-11e9-9ca0-5c2a63d2b90e.png)